src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Configuration.php

```diff
@@ -119,6 +125,11 @@ private function __construct(array $thresholds = [], $regex = '', $verboseOutput
                 throw new \InvalidArgumentException(sprintf('The baselineFile "%s" does not exist.', $this->baselineFile));
             }
         }
+
+        if (file_exists($logFile)) {
+            throw new \InvalidArgumentException(sprintf('The log file "%s" already exists.', $logFile));
```

Comment:
Why fail here? this looks fine to me. We could even append, wdyt?

Reply:
Ok, we append now (for my taste I wouldn't use it like this because the log file is maybe harder to read if it is appended but yes, now it's possible and yes, it doesn't fail now)